### ISSUE TITLE
Explore automating mapping to status platforms

### ISSUE THREAD

**Issue by tidoust** [w3c/browser-statuses#5]
The data from Web platform status sites usually include links to the spec. This URL should typically match the nightly or the release URL in browser-specs, allowing to:

1. Validate the mappings that already exist
2. Pre-populate the mappings table automatically when a new spec is added to the list.

**Comment by tidoust** [comment]
Status update: tools are now available to propose additional mappings and create pull requests for them automatically. #13 is an example of PR that can be automatically created.

- I validated existing mappings based on this, adding `manual` flags where ever needed: https://github.com/w3c/browser-statuses/commit/a203a0604055a6ee4a35e777486963b647dd1c77
- Data files were completed to incorporate new mappings: https://github.com/w3c/browser-statuses/commit/48fe47ec205016c64831412b2b003ce3bd5af0f0
- Additional data files were created for browser-specs for which we did not yet have any mapping: https://github.com/w3c/browser-statuses/commit/570ae6c94f12e5d214afc35742d539680d4787f9

Still todo:
- create a job to periodically run the PR creation process
- create a job to periodically validate manual data
- create a job to periodically identify and review "representative" mappings.
